spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+RSpec.describe 'Renders the post show page', type: :feature do
+  before :each do
+    @user = User.create(name: 'John Carson', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'I am an auditor with 30 years working experience.', posts_counter: 0)
+    @first_post = Post.create(title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0,
+                              author_id: @user.id)
+    @second_post = Post.create(title: 'welcome', text: 'This is my second post', comments_counter: 0, likes_counter: 0,
+                               author_id: @user.id)
+    #   5.times do |_i|
+    #     Comment.create(text: 'Nice post!!', author_id: @user.id, post_id: @first_post.id)
+    #   end
+    @comment = Comment.create(text: 'Nice post!!', author_id: @user.id, post_id: @first_post.id)
+    visit user_post_path(@first_post.author, @first_post)
+  end
+
+  scenario 'displays the post title' do
+    expect(page).to have_content(@first_post.title)
+  end
+
+  scenario 'shows the person who wrote the post' do
+    expect(page).to have_content('John Carson')
+  end
+
+  it 'can see the username of each commentor.' do
+    expect(page).to have_content('John Carson')
+  end
+
+  scenario 'displays number of comments' do
+    expect(page).to have_content(@comment.text)
+  end
+
+  scenario 'displays number of likes' do
+    expect(page).to have_content('Likes: 0')
+  end
+
+  scenario 'displays the content of the post' do
+    expect(page).to have_content(@first_post.text)
+  end
+
+  scenario 'can see the comments of each commentor.' do
+    expect(page).to have_content 'Nice post!!'
+    expect(page).to have_content 'Nice post!!'
+    expect(page).to have_content 'Nice post!!'
+    expect(page).to have_content 'Nice post!!'
+    expect(page).to have_content 'Nice post!!'
+  end
+end
```

Comment:
- [ ] [REQUIRED] We have 4 tests failing for the `spec/requests/posts_spec.rb` and the `spec/requests/users_spec.rb:`. The `request` directory is not part of your file changes so I decided to attach them here.  Kindly fix the test errors before asking for another review.
<img width="817" alt="Screenshot 2022-11-01 at 20 31 14" src="https://user-images.githubusercontent.com/58692590/199325458-da3439dc-53f1-40c9-912f-fbc1518aedff.png">

<img width="870" alt="Screenshot 2022-11-01 at 20 37 11" src="https://user-images.githubusercontent.com/58692590/199325479-a7e33bbe-939b-4ce1-893e-d71873b10a6c.png">

<img width="868" alt="Screenshot 2022-11-01 at 20 36 26" src="https://user-images.githubusercontent.com/58692590/199325515-74a0289c-2760-4e6d-88ca-d6471b85d3e7.png">

<img width="807" alt="Screenshot 2022-11-01 at 20 34 52" src="https://user-images.githubusercontent.com/58692590/199325585-beea183c-fcd1-4627-9ab0-a40b14fd5e4d.png">

<img width="448" alt="Screenshot 2022-11-01 at 20 52 15" src="https://user-images.githubusercontent.com/58692590/199325969-0b05364f-c532-49c5-ad86-e1c7fd6c335a.png">